libflux/include/influxdata/flux.h

```diff
@@ -29,9 +29,9 @@ void flux_free(void *);
 // flux_ast_pkg_t is the AST representation of a flux query as a package.
 struct flux_ast_pkg_t;
 
-// flux_parse will take in a string and return the AST representation
-// of the query.
-struct flux_ast_pkg_t *flux_parse(const char *);
+// flux_parse will take in a file name string and a source string then
+// return the AST representation of the query.
+struct flux_ast_pkg_t *flux_parse(const char *, const char *);
```

Comment:
It would be good to add parameter names here to aid documentation.

Reply:
Done